CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased Major]
 
+### Added
+- Implement the possibility to use `FromPython` to generate rotation and position in the python
+  bindings starting from arrays (https://github.com/robotology/idyntree/pull/959)
```

Comment:
```suggestion
  SWIG bindings starting from arrays (https://github.com/robotology/idyntree/pull/959)
```

As we have both SWIG and pybind11 bindings, better to be unambigous in releases notes.